src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -225,19 +230,30 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
         [ValidateRange(0, 65535)]
         [Parameter(Mandatory = true, ParameterSetName = TcpPortParameterSet)]
         public int TcpPort { get; set; }
-
+        
```

Comment:
Please remove extra spaces - turn on auto truncating in your editor.

Reply:
I've removed any extra whitespace, thank you for the tip!